lib/protocol/src/openctx-protocol.schema.ts

```diff
@@ -62,12 +76,38 @@ export interface Selector {
      */
     contentContains?: string
 }
-export interface ItemsParams {
+/**
+ * A mention contains presentation information relevant to a resource.
+ */
+export interface Mention {
+    /**
+     * A descriptive title.
+     */
+    title: string
+    /**
+     * A URI for the mention item.
+     */
+    uri: string
+    data?: {
+        [k: string]: unknown | undefined
+    }
+}
+export interface MentionsParams {
     /**
      * A search query that is interpreted by providers to filter the items in the result set.
      */
     query?: string
 }
+export interface ItemsParams {
+    /**
+     * A message that is interpreted by providers to return relevant items.
```

Comment:
Just to check, in practice for our cody integration we will set both fields? IE message will be the final chat input and mention will what was selected from the mention dropdown?

I wonder if we should be clearer on the docstring here. For example in all our @ mention providers (except packages) we don't actually read this field and just rely on mention.

```suggestion
     * A message that can be interpreted by providers to return relevant items.
```